content/applications/hr/time_off.rst

```diff
@@ -481,8 +500,8 @@ time off. This allows users to see which employees are taking time off, how much
 taking, and what time off types are being used.
 
 Any report can be added to a spreadsheet through the *Insert in Spreadsheet* button that appears in
```

Comment:
I am only seeing this button appear on the 'by Type' reporting page... it's definitely worth a double-check to ensure the accuracy of this statement during your revisions. Thanks! 👍

Reply:
It appears for the 'by Employee' report if you change to a graph view- I'll add that!